src/open_inwoner/js/django-admin.js

```diff
@@ -1,3 +1,4 @@
 import './admin/ckeditor'
+import './admin/admin-overrides/admin-overrides'
```

Comment:
> And import it in a `index.js` file (needs to be created)

Then this could be simplified

```suggestion
import './admin/admin-overrides'
```